v1/artifact.go

```diff
@@ -25,7 +25,7 @@ type ArtifactVersion struct {
 	Tag      string            `json:",omitempty"`
 	Type     string            `json:"artifact_type"`
 	ID       string            `json:"id"`
-	Version  int               `json:"version"`
+	Version  string            `json:"version"`
```

Comment:
Also one thing to note, in testing I've seen that the `atlas_artifact` [slug](https://www.terraform.io/docs/providers/atlas/r/artifact.html#slug) attribute returns `username/artifact/archive/version` when I thought in Atlas we refer to a slug as `username/artifact`.

For example, I echo'd an `atlas_artifact`'s slug in Terraform and it output `bensojona/potomac-hello-world/archive/3`.